style.md

```diff
@@ -309,10 +309,10 @@ private val defaultCharset: Charset? =
         EncodingRegistry.getInstance().getDefaultCharsetForPropertiesFiles(file)
 ```
 
-Properties declaring a `get` and/or `set` function should place each on their only line with a normal indent (+4). Format them using the same rules as functions.
+Mutable properties declaring a `get` and/or `set` function should place each on their only line with a normal indent (+4). Format them using the same rules as functions.
```

Comment:
This needs reverted, as
```kotlin
val foo: String
    get() = "Hi"
```
is not mutable.

Reply:
Fixed